src/components/Modals/ContactCourseModal.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useContext, useState } from 'react';
+import { message as notif } from 'antd';
+import { CourseParticipant, ParsedCourseOverview } from '../../types/Course';
+import DialogModalBase from './DialogModalBase';
+import Images from '../../assets/images';
+import { ApiContext } from '../../context/ApiContext';
+import { ModalContext } from '../../context/ModalContext';
+
+interface Props {
+  course: ParsedCourseOverview;
+  selectedParticipants: CourseParticipant[];
+  setSelectedParticipants: (selectedParticipants: CourseParticipant[]) => void;
+  setSelectingParticipants: (isSelecting: boolean) => void;
+  type: 'instructorToParticipants' | 'participantToInstructors';
+}
+
+const ContactCourseModal: React.FC<Props> = ({
+  course,
+  setSelectedParticipants,
+  selectedParticipants,
+  type,
```

Comment:
I'm not quite sure if this should really be one component, especially as for the "participantToInstructors" the selectedParticipants is useless. Maybe it makes sense to split this up in three components, `<ContactForm title=... description=... submit={() => ...} />` and then two wrappers for that for the two usecases.